cinema/tests/test_movie_session_api.py

```diff
@@ -117,9 +104,7 @@
         response = self.client.get("/api/cinema/movie_sessions/1/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data["movie"]["title"], "Titanic")
-        self.assertEqual(
-            response.data["movie"]["description"], "Titanic description"
-        )
+        self.assertEqual(response.data["movie"]["description"], "Titanic description")
```

Comment:
This assertion verifies that the movie's description is correctly retrieved, ensuring the API returns the expected data structure.